### ISSUE TITLE
Adjust the size of window

### ISSUE THREAD

**Issue by baibanbao** [vincelwt/chatgpt-mac#102]
Would you please consider to add a feature to adjust the size of window?

It would be nice to drag and resize the window and make the contents on this app more readable.

**Comment by bb1218** [comment]
great! hope so too

**Comment by Globolo001** [comment]
@baibanbao Already possible. Just use your mouse pointer to the window sides

**Comment by edgarchm** [comment]
> @baibanbao Already possible. Just use your mouse pointer on the window sides

Nope, it is not. I'm pointing all sides and corners, no resize option.

**Comment by Globolo001** [comment]
Maybe you got an old version? Works for me lol

<img width="756" alt="Screenshot 2023-05-20 at 12 47 18" src="https://github.com/vincelwt/chatgpt-mac/assets/58426549/f99fd3df-421a-4fd0-ad8a-62a8ae84357b">

**Comment by rizerzero** [comment]
For me, it works, but after closing the app it loses the size, so it would be better to have it a setting.

**Comment by mahdemir** [comment]
You can adjust the default window size in the index.js file inside the application at this path: 
``` bash
 /Applications/ChatGPT.app/Contents/Resources/app/index.js
```
Change the values by line 34 and 35 and you're done 🎉 

<img width="1175" alt="gpt_default_window_size" src="https://github.com/vincelwt/chatgpt-mac/assets/103860270/2676d396-1ce4-4128-8773-f7b78a84742e">
